spec/models/member_spec.rb

```diff
@@ -0,0 +1,19 @@
+# == Schema Information
+#
+# Table name: members
+#
+#  id         :integer          not null, primary key
+#  created_at :datetime
+#  updated_at :datetime
+#  last_name  :string(255)
+#  first_name :string(255)
+#  email      :string(255)
+#
+
+require 'rails_helper'
+
+RSpec.describe Member, :type => :model do
```

Comment:
Could you use the `type: :model` syntax? I thought rubocop should have caught this?